app/serializers/event_schedule_serializer.rb

```diff
@@ -26,8 +26,7 @@ class EventScheduleSerializer < ActiveModel::Serializer
   attributes :date, :room
 
   def date
-    t = object.start_time
-    t.blank? ? '' : %( #{I18n.l t, format: :short}#{t.formatted_offset(false)} )
+    object.start_time&.change(zone: object.event.program.conference.timezone)
```

Comment:
I am not 100% of the implications of this change and the various timezone features.